### PR TITLE
Flyway task done

### PR DESCRIPTION
**Pull request by FedorSerg**

### CODE REVIEW

**Review by goponenkoev — CHANGES_REQUESTED** [review]
Plz update ReadMe.md to get how to work with Flyway in future

File: src/main/resources/db/migration/V1__Create_db.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
+create table DRIVER
```
Comment (goponenkoev) [inline src/main/resources/db/migration/V1__Create_db.sql:1]: Do u implement the connection between tables via foreign key and primary key

**Comment by DeryckSPBU** [comment]
Not the best files naming V1** V1**, it's better to name the first file like V1__*smth here*.sql, second one V2__*smth here*.sql and so on, to tell FlyWay in which order invoke these files

**Review by goponenkoev — CHANGES_REQUESTED** [review]
Did u run Gradle task `flywayMigrate` or u just believe in your code?
I've tried to run it. And got failed. U forgot about two points, try to execute and fix it plz

File: src/main/resources/db/migration/V1__Create_db.sql (hunk 1 above)
Comment (FedorSerg) [reply src/main/resources/db/migration/V1__Create_db.sql:1]: done

Comment (goponenkoev) [reply src/main/resources/db/migration/V1__Create_db.sql:1]: I cann't see find any changes, could u show me?

**Review by goponenkoev — APPROVED** [review]

File: src/main/resources/db/migration/V1__Create_db.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    LICENSE   varchar(10) not null,
+    PHONE_NUM varchar(11),
+    LOGIN     varchar(64) not null,
+    PASSWORD  varchar(64) not null
+);
+
+create table TICKET
+(
+    ID        int     not null primary key,
+    ID_CLIENT int     not null references DRIVER (ID),
```
Comment (DeryckSPBU) [inline src/main/resources/db/migration/V1__Create_db.sql]: You called table as DRIVER, why is there ID_CLIENT? also in the definition of table OFFER you use CLIENT_ID, confusing.
It's good practice to have everything in one style

File: src/main/resources/db/migration/V1__Create_db.sql [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+);
+
+create table OFFER
+(
+    ID         int         not null primary key,
+    CLIENT_ID  int         not null references DRIVER (ID),
+    ADMIN_ID   int         not null references ADMINISTRATOR (ID),
+    CAR_ID     int         not null references CAR (ID),
+    PAYMENT    int         not null,
+    BEGIN_DATE varchar(10) not null,
```
Comment (DeryckSPBU) [inline src/main/resources/db/migration/V1__Create_db.sql]: Probably it's better to use here timestamp datatype, maybe timestamp without time zone, also default value or even trigger might be good here, to fill this cell automatically

File: src/main/resources/db/migration/V1__Create_db.sql [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+(
+    ID     int         not null primary key,
+    MARQUE varchar(16),
+    MODEL  varchar(16),
+    NUMBER varchar(10) not null
+);
+
+create table OFFER
+(
+    ID         int         not null primary key,
```
Comment (DeryckSPBU) [inline src/main/resources/db/migration/V1__Create_db.sql]: If we use surrogate key like ID, it's worth to create sequences or use type serial to autoincrement this value, we are definitely do not want to remember which one ID we should set next to insert it into database.
Apply it everywhere please

File: src/main/resources/db/migration/V2__Add_informtion.sql [hunk 5]
```diff
@@ -0,0 +1,26 @@
+insert into DRIVER
+values (1, 'Gwion Ritter', '1234567890', '142359708', '89112345678', 'login', 'password'),
```
Comment (DeryckSPBU) [inline src/main/resources/db/migration/V2__Add_informtion.sql]: After you add serial or sequence for ID column, we don't need to set ID explicitly, so remove them

**Review by DeryckSPBU — CHANGES_REQUESTED** [review]

**Review by DeryckSPBU — APPROVED** [review]

**Review by goponenkoev — APPROVED** [review]

**Pull request merged**